network/src/peers/peers.rs

```diff
@@ -290,12 +290,56 @@ impl<S: Storage + Send + Sync + 'static> Node<S> {
 
     pub(crate) async fn send_peers(&self, remote_address: SocketAddr) {
         // Broadcast the sanitized list of connected peers back to the requesting peer.
-        let peers = self
-            .peer_book
-            .connected_peers()
-            .into_iter()
-            .filter(|&addr| addr != remote_address)
-            .choose_multiple(&mut rand::thread_rng(), crate::SHARED_PEER_COUNT);
+        //
+        // Order of tried sets for bootnode peer lists:
```

Comment:
> bootnode peer lists

the logic below applies to all peers - only the second branch is bootnode-specific